invenio_app_ils/ui/src/invenio_app_ils/common/api/loan.js

```diff
@@ -65,10 +67,24 @@ const fetchLoans = (
   return http.get(`${loanURL}?q=${qs}&sort:${sortByOrder}`);
 };
 
+const viewDetailsClickUrl = loanPid => {
```

Comment:
I think the API layer shouldn't know anything about UI, like clicks or urls or views.
Why do you want to put this here? Probably better in `urls.js`, for example?